apps/web/src/components/features/search/SearchBar.tsx

```
@@ -0,0 +1,166 @@
+                onChange={handleLocationChange}
+                className="border-0 p-0 focus-visible:ring-0 flex-1"
+              />
+            </div>
+
+            {showSuggestions && (
+              <ul 
+                className="absolute w-full mt-1 bg-background border rounded-md shadow-md z-20 max-h-48 overflow-y-auto"
+                role="listbox"
+                aria-label="Location suggestions"
+              >
+                {suggestions.map((suggestion) => (
```

Biome (1.9.4)

[error] The HTML element with the interactive role "listbox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "listbox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)